src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -1,22 +1,22 @@
 
 package tech.cryptonomic.conseil
 
+import akka.Done
 import com.typesafe.config.ConfigFactory
 import com.typesafe.scalalogging.LazyLogging
-import tech.cryptonomic.conseil.tezos._
+import tech.cryptonomic.conseil.tezos.{FeeOperations, TezosNodeInterface, TezosNodeOperator, TezosDatabaseOperations => TezosDb, _}
```

Comment:
How come you use this alias here?

Reply:
I found it noisy at usage site

I'll revert if it's confusing

Reply:
Generally, we try to avoid aliasing but we can see your point.